.github/workflows/test.yml

```diff
@@ -101,14 +101,18 @@ jobs:
         with:
           target-ros2-distro: ${{ matrix.ros_distribution }}
           vcs-repo-file-url: "${{ github.workspace }}/.github/workflows/repo_file_for_test_cpp_package.repos"
-        # TODO(tmoulard): re-enable this test once it passes on Windows
-        if: runner.os != 'Windows'
       - run: test -d "${{ steps.test_all_packages_in_repo.outputs.ros-workspace-directory-name }}/install/osrf_testing_tools_cpp"
-        # TODO(tmoulard): re-enable this test once it passes on Windows
-        if: runner.os != 'Windows'
+        name: "Check that ament_copyright install directory is present (Non-Windows only)"
+        if: matrix.os != 'windows-latest'
+      - run: test -d "${{ steps.test_all_packages_in_repo.outputs.ros-workspace-directory-name }}/install/share/osrf_testing_tools_cpp"
```

Comment:
This pr changes windows to use the `--merge-install` option (as suggested in the ros2 wiki). This test step had to be modified since `--merge-install` puts all the pkgs in a shared install space instead of subfolders.